template/build/vue-loader.conf.js

```diff
@@ -1,7 +1,7 @@
 'use strict'
 const utils = require('./utils')
 const config = require('../config')
-const isProduction = process.env.NODE_ENV === 'production'
+const isProduction = process.env.NODE_ENV === 'production' || process.env.VUE_E2E === 'test'
```

Comment:
See `/test/e2e/runner.js`: 

There we set `NODE_ENV` to `test`, but we use the wbepack dev server in the `webpack.prod.conf.js`, because the want to run e2e tests against a devserver that serves likes like in production. Therefore, we have to use a second env variable as a workaround to make the production config behave like `NODE_ENV === 'production'`